service/eventer/github/github.go

```diff
@@ -108,7 +108,6 @@ func (e *Eventer) FetchContinuously(projects []string, environment string) (<-ch
 	ticker := time.NewTicker(e.pollInterval)
 
 	go func() {
-		// TODO this map just grows which means we fill the programm with memory.
```

Comment:
This is actually not true. Back then I got it wrong. The etag map is indexed by the configured project names. These are finite. All good. Just removing this useless TODO.